extensions/blocks/contact-info/style.native.scss

```
@@ -0,0 +1,26 @@
+@import "~@wordpress/base-styles/colors";
```

Could you use your import like this, to stay consistent with the rest of the blocks?

```suggestion
@import '../../shared/styles/gutenberg-base-styles.scss';
```